CMakeLists.txt

```diff
@@ -38,3 +38,10 @@ add_executable(tmatrix ${tmatrix_SRC})
 include_directories(include)
 
 install(TARGETS tmatrix DESTINATION bin)
+
+# Completions
+install(FILES extra/completions/_tmatrix
```

Comment:
I think we can name them tmatrix-completion.bash and tmatrix-completion.zsh and then renamed them appropriately in the final directory. The repo directory can just be `completion` instead of `extra/completions`.
Move the closing parentheses into the next row.